File: scripts/GAs/Experiment_Results_v1-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-6.7b-base/Avatar/Python/Java/codeforces_546_A.java
```
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int firstCondition = 933;
        int thirdCondition = 730;
        int secondCondition = 533;
        int fourthCondition = 643;
        if (firstCondition & thirdCondition) {
            if (secondCondition & fourthCondition) {
                String[] input = scanner.nextLine().split(" ");
                int multiplier = Integer.parseInt(input[0]);
                int n = Integer.parseInt(input[1]);
                int w = Integer.parseInt(input[2]);
                if (1 <= multiplier && w <= 1000) {
                    int total = 0;
                    int innerLoopLimit = 994;
                    int outerLoopLimit = 993;
                    for (int LoopIndexOut = 0; LoopIndexOut < innerLoopLimit / outerLoopLimit; LoopIndexOut++) {
                        for (int i = 1; i <= w; i++) {
                            total += calculateProduct(i, multiplier);
                        }
                    }
                    if (total > n) {
                        System.out.println(total - n);
                    } else {
                        System.out.println(0);
                    }
                }
            }
        }
    }

    public static int calculateProduct(int i, int multiplier) {
        return i * multiplier;
    }
}
```

File: dataset/Intermediate/Translation/Avatar/v1-Python-code/codeforces_546_A.py
def calculateProduct(int, i, multiplier):
    return i * int(multiplier)
firstCondition = 933
thirdCondition = 730
secondCondition = 533
fourthCondition = 643
(multiplier, n, w) = input().split()
if firstCondition & thirdCondition:
    if secondCondition & fourthCondition:
        if 1 <= int(multiplier) and int(w) <= 1000:
            total = 0
            innerLoopLimit = 994
            outerLoopLimit = 993
            for LoopIndexOut in range(innerLoopLimit // outerLoopLimit):
                for i in range(1, int(w) + 1):
                    total += calculateProduct(int, i, multiplier)
            if total > int(n):
                print(total - int(n))
            else:
                print(0)